classes/FaviconController.php

```diff
@@ -22,26 +21,36 @@ public function index()
 
         $finalPath = $inputPath = storage_path('app/media' . $settings->favicon);
 
-        if ($settings->favicon_16) {
+        return response()->file($finalPath, [
+            'Content-Type' => 'image/x-icon',
+        ]);
+    }
 
-            $destinationPath = storage_path('app/initbiz/seostorm/favicon/' . dirname($settings->favicon) . '/');
-            $finalPath = $outputPath = $destinationPath . basename($settings->favicon);
+    public function generateManifest()
+    {
+        $settings = Settings::instance();
+        $responseArray = [];
+        
+        if (!$settings->webmanifest_enabled) {
+            $controller = new Controller();
```

Comment:
I'm pretty certain that returning 404 from this method doesn't make sense. Does it?